File: WebApi/Models/LiveMatches/FeaturedGameInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MatchStatistic.WebApi.Models.Matches
{
	public class FeaturedGameInfo
	{
		public string GameMode { get; set; }
		public long GameLength { get; set; }
		public long MapId { get; set; }
		public string GameType { get; set; }
		public BannedChampion[] BannedChampions { get; set; }
		public long GameId { get; set; }
		public Observer Observers { get; set; }
		public long GameQueueConfigId { get; set; }
		public long GameStartTime { get; set; }
		public Participant[] Participants { get; set; }
		public string PlatformId { get; set; }
	}
}

File: ClientApp/src/app/models/LiveMatches/featuredGameInfo.ts
import { BannedChampion } from "./bannedChampions.model";
import { Observer } from "./observer.model";
import { Participant } from "./participant.model";

export class FeaturedGameInfo {
  GameMode: string;
  GameLength: number;
  MapId: number;
  GameType: string;
  BannedChampions: BannedChampion[];
  GameId: number;
  Observers: Observer;
  GameQueueConfigId: number;
  GameStartTime: number;
  Participants: Participant[];
  PlatformId: string;
}
